user_docs/en/changes.t2t

```diff
@@ -692,6 +692,7 @@ Highlights of this release include  support for charts in Microsoft word and Pow
 == Changes ==
 - The HIMS Braille Sense/Braille EDGE/Smart Beetle and Hims Sync Braille display drivers have been replaced by one driver. The new driver will automatically be activated for former syncBraille driver users. (#7459) 
  - Some keys , notably scroll keys, have been reassigned to follow the conventions used by Hims products. Consult the user guide for more details.
+- F1 can now be used in most NVDA dialogs to get context sensitive help.

```

Comment:
I  believe Change log entry is added just before merge most of the time. It should also be in the section for NVDA 2020.3 or whatever is the most recent version at the time of merge, not in the section for 2018.3 as it is now.